src/components/smartSearch/inputs/StyledTagSelect.tsx

```diff
@@ -0,0 +1,52 @@
+import { Theme } from '@material-ui/core';
+import { Autocomplete, AutocompleteProps } from '@material-ui/lab';
+import { Chip, makeStyles } from '@material-ui/core';
+
+import StyledTextInput from './StyledTextInput';
+import { ZetkinTag } from 'types/zetkin';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+    autocomplete: {
+        display: 'inline',
+    },
+    input: {
+        lineHeight: theme.typography.h4.lineHeight,
+        verticalAlign: 'middle',
+        width: 'auto',
+    },
+}));
+
+type StyledTagSelectProps = Omit<AutocompleteProps<ZetkinTag, true, undefined, false>, 'renderInput'>
+
+const StyledTagSelect = (props: StyledTagSelectProps): JSX.Element => {
+    const classes = useStyles();
+    return (
+        <Autocomplete
```

Comment:
I feel like the `Autocomplete` isn't working great. Maybe it's enough to just make it wider by default?

![image](https://user-images.githubusercontent.com/550212/128151145-4862aaf9-80a7-4551-af7e-bf5cb3ad93d4.png)